### PR TITLE
Added drinking glasses and bottles to the autolathe

### PR DESCRIPTION
**Pull request by VTCobaltblood**
changes: 
  - tweak: "Added drinking glasses and bottles to the autolathe. Brought illegal items back to the bottom of the list."

A drinking glass costs 300 glass (as opposed to 500 for beakers). Bottles cost 800.

### CODE REVIEW

**Comment by VTCobaltblood** [comment]
!review

File: code/modules/reagents/reagent_containers/food/drinks/bottle.dm [hunk 1]
```diff
@@ -12,6 +12,7 @@
 	force = 5
 	var/smash_duration = 5 //Directly relates to the 'weaken' duration. Lowered by armor (i.e. helmets)
 	var/isGlass = 1 //Whether the 'bottle' is made of glass or not so that milk cartons dont shatter when someone gets hit by it
+	matter = list("glass" = 800)
```
Comment (Sindorman) [inline code/modules/reagents/reagent_containers/food/drinks/bottle.dm]: Small math inconsistency. Glass holds 30 volume and is 300, hence it is 10 glass per 1 volume. But this holds 100 and cost is 800 which means it is 8 glass per 1 volume

Meaning it should be 1000

Comment (VTCobaltblood) [reply code/modules/reagents/reagent_containers/food/drinks/bottle.dm]: The beaker and glass are also inconsistent between each other. You're free to headcanon that the drinking glass has a thicker surface. I think the cost is fine.

Comment (Sindorman) [reply code/modules/reagents/reagent_containers/food/drinks/bottle.dm]: Aren't bottles plastic?

Comment (Alberyk) [reply code/modules/reagents/reagent_containers/food/drinks/bottle.dm]: Regular bottles are glass, as they can leave glass shards when you throw, I think.

**Pull request merged**